packages/comms/src/services/wsLogaccess.ts

```diff
@@ -199,7 +199,7 @@ export class LogaccessService extends LogaccessServiceBase {
                 }
                 return {
                     lines: lines,
-                    total: response.TotalLogLinesAvailable || 10000
+                    total: response.TotalLogLinesAvailable ?? 0
```

Comment:
If the total number of log lines is unknown (undefined) it does not mean, there are none...  I think we should keep the 10000 to prevent any compatibility issues.